README.md

```diff
@@ -1,207 +1,339 @@
 
-## Running
-Enter `lukso start` to start a node.
+> 🛠️ Work In Progress, available soon.
+
+## Installing LUKSO
+
+```bash
+# 1. The command below installs the LUKSO client and prompts the user with default settings to get started as a validator
+$ curl https://install.lukso.network | sh
+
+# 2. You will need to agree to the lukso-cli terms before continuing. Simply type "Y" to agree
+https://github.com/lukso-network/tools-lukso-cli/blob/main/TERMS.md
+
+# 3. Create a working folder where you want your clients to store their data:
+$ mkdir myLUKSOFolder && cd ./myLUKSOFolder
+
+# 4. This command downloads the network configs from
+# https://github.com/lukso-network/network-configs to your recently created "myLUKSOFolder/configs"
+# It will not overwrite any existing config folders, data or keystore the user might have
+$ lukso init
+
+# 5. Install your desired clients using
+$ lukso install
+# Or simply type to accept all Terms & Conditions of LUKSO's clients.
+# You don't need to run this if you already installed LUKSO.
+$ lukso install --agree-terms
+
+```
 
 ## Available parameters
+
 `lukso <command> [geth, prysm, validator, *all*] [--flags]`
-> *all* means that you can skip an argument for all possible options to run (default, only option for download)
+
+> _all_ means that you can skip an argument for all possible options to run (default, only option for download)
 
 | Command        | Description                                  |
-|----------------|----------------------------------------------|
+| -------------- | -------------------------------------------- |
 | init           | Initializes configuration files              |
-| start          | Starts up all or specific client(s)          |
-| stop           | Stops all or specific client(s)              |
+| install        | Downloads all default client(s)              |
 | log            | Show logs                                    |
-| status         | Shows status of all or specified client      |
 | reset          | Resets data directories                      |
-| install        | Downloads all default client(s)              |
+| start          | Starts up all or specific client(s)          |
+| status         | Shows status of all or specified client      |
+| stop           | Stops all or specific client(s)              |
 | update         | sets client(s) to desired version            |
 | validator      | Manages validator-related commands           |
-| validator init | Initializes your validator with deposit keys | 
+| validator init | Initializes your validator with deposit keys |
+| version        | Display version of LUKSO CLI                 |
 
+## init
 
-### Start your node
+This command downloads the network configs from https://github.com/lukso-network/network-configs to your recently created "myLUKSOFolder/configs"
+It will not overwrite any existing config folders, data or keystore the user might have
+
+```bash
+
+$ lukso init
```

Comment:
We can add a comment here so that user knows that he needs to be inside his working folder - the description sort of assumes that `init` does this automatically in a `myLUKSOFolder`, but this is not true